contrib/JLightning/src/de/renepickhardt/ln/JLightningRpc.java

```diff
@@ -0,0 +1,147 @@
+package de.renepickhardt.ln;
+/**
+ * JLightningRpc extends the UnixDomainSocketRpc and exposes the specific 
+ * API that is provided by c-lightning. It  is a java client library for the 
+ * c-lightning node. It connects to c-lightning via a Unix Domain Socket over
+ * JsonRPC v2.0 
+ * 
+ * This file is basically a port of the pylightning python client library
+ * that comes with c-lightning.
+ * 
+ * The Author of this Java Client library is Rene Pickhardt. 
+ * He also holds the copyright of this file. The library is licensed with
+ * a BSD-style license. Have a look at the LICENSE file. 
+ * 
+ * If you like this library consider a donation via bitcoin or the lightning
+ * network at http://ln.rene-pickhardt.de
+ * 
+ * @author Rene Pickhardt
+ */
+
+import java.util.HashMap;
+
+
+
+public class JLightningRpc extends UnixDomainSocketRpc {
+
+	public JLightningRpc(String socket_path) {
+		super(socket_path);
+		
+	}
+	
+	/**
+	 * Delete unpaid invoice {label} with {status}
+	 * @param label of the invoice
+	 * @param status status of the invoice
+	 * @return
+	 */
+	public String delInvoice(String label, Status status) {
+		HashMap<String, String> payload = new HashMap<String,String> ();
+		payload.put("label", label);
+		payload.put("status", status.toString());
```

Comment:
To handle `NullPointerException` here on `toString()`, you can use [`String.valueOf(status)`](https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#valueOf-java.lang.Object-). In case of `null`, it will return the `String` `"null"`.